src/components/utils/useReducedMotion/useReducedMotion.stories.mdx

```diff
@@ -0,0 +1,42 @@
+import {ArgsTable, Meta, Story, Canvas} from '@storybook/addon-docs';
+import Accordion from '../../accordion/Accordion';
+import AccordionItem from '../../accordion/AccordionItem';
+
+<Meta title="Utilities/useReducedMotion" />
```

Comment:
Currently Utilities is mainly for css level utilities. We're going to review the whole documentation structure during the curent storybook but I maybe for this hook we can use `Accessibility/useReducedMotion`

@brainly/design-infra wdyt guys?

Reply:
cc @katarzynatobis @domidomi @clxandstuff

Reply:
@przytrzask you can leave it as it is for now. We will change it to more appropriate group once we internally finalise the structure.

Reply:
@przytrzask You can leave the documentation content as is. We'll take care of moving things to the appropriate sections separately once we adjust the structure. On your part I would just ask you to follow this comment https://github.com/brainly/style-guide/pull/2401#discussion_r860842619